main.py

```diff
@@ -139,6 +139,10 @@ def __getitem__(self, key):
     if e not in cattypes:
         cattypes.append(e)
 
+# function to autocomplete choices for /gift, /giftcat, and /forcespawn, which also allows more than 25 options
+async def choicer(interaction: discord.Interaction, current: str) -> list[discord.app_commands.Choice[str]]:
+    return [app_commands.Choice(name=choice, value=choice) for choice in cattypes if current.lower() in choice.lower()][:25]
```

Comment:
shouldn't this be `return [discord.app_commands...]`

Reply:
uhh probably but it still works for some reason